gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanIntegrationTestLauncher.java

```diff
@@ -0,0 +1,52 @@
+/* (c) 2014 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.azkaban;
+
+import java.io.File;
+
+import org.apache.log4j.Logger;
+
+import azkaban.jobExecutor.AbstractJob;
+
+import gobblin.test.setup.config.TestHarnessLauncher;
+
+/**
+ * This class launches the TestHarness framework in Gobblin and executes tests using TestNG
+ *
+ * Created by spyne on 6/8/15.
+ */
+public class AzkabanIntegrationTestLauncher extends AbstractJob {
+  private static final String CONFIG_PATH = "testharness.gobblin.azkaban.properties";
+  private static final Logger LOG = Logger.getLogger(AzkabanIntegrationTestLauncher.class);
+  private static final String id = "gobblin.idpc.integration.test";
```

Comment:
Can we remove the idpc specific configs? As discussed you should be able to use the `AzkabanIntegrationTestLauncher(String jobId, Properties props)` to get all the relevant configuration properties. The Azkaban documentation should have that feature listed somewhere, if not I'm sure we can ask azkaban-dev.